src/PrestaShopBundle/Form/Admin/Sell/Discount/CartConditionsType.php

```diff
@@ -79,6 +83,26 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ),
                 ],
             ])
+            ->add(self::SPECIFIC_PRODUCTS, ProductSearchType::class, [
+                'layout' => EntitySearchInputType::LIST_LAYOUT,
+                'entry_type' => PackedProductType::class,
```

Comment:
This is not the entry type adapted for this use case Although it is very similar editing pack products and discount conditions is not the same use case A new entry type must be created for this specific use case, for this PR it will look like the `PackedProductType` a lot because we only have an additional quantity input

But in a future PR we'll have to handle product combination selection, and this new feature will be added in this new dedicated entry type

The new form type can be named `PrestaShopBundle\Form\Admin\Sell\Discount\ProductConditionType` for example